packages/core-app-api/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Patch Changes
 
+- 1ff062c169: Add optional scope argument to OAuth2.signIn method
```

Comment:
You don't want to edit this file directly, you want to create a changeset as documented here: https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets